test/sg.js

```diff
@@ -5,18 +5,22 @@ import sg from '../src/sg';
 import {
     call,
     cps,
-    fork,
-    spawn,
-    put,
-    take,
-    takeEvery,
-    takeLatest,
-    cancel,
     delay,
-    race,
-    join,
+    fork,
 } from '../src/effects';
 
+// const {
+//     race,
+//     join,
+//     fork,
+//     spawn,
+//     put,
+//     take,
+//     takeEvery,
+//     takeLatest,
+//     cancel,
+// } = events;
```

Comment:
commented out because you splitted the prs ? Will be reintroduced in later prs ?